packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -193,14 +193,14 @@ describe('common/sqlite', () => {
-                                                                                                                                                                                                    `)
+                                                                                                                                                                                                                                                                                                                    // *** WARNING ***
+                                                                                                                                                                                                                                                                                                                    // 
+                                                                                                                                                                                                                                                                                                                    // These models were enriched with \`@@map\` information taken from the previous Prisma schema.
+                                                                                                                                                                                                                                                                                                                    // - Model "AwesomeNewPost"
+                                                                                                                                                                                                                                                                                                                    // - Model "AwesomeProfile"
+                                                                                                                                                                                                                                                                                                                    // - Model "AwesomeUser"
+                                                                                                                                                                                                                                                                                                                    // 
+                                                                                                                                                                                                            `)
```

Comment:
I hate these snapshot diffs. 
Do we have an issue in tech debt about this that is very high on the list to fix?

Reply:
Yes :)

Reply:
Thank you.